### ISSUE TITLE
set data attributes on rows

### ISSUE THREAD

**Issue by ctcpip** [tubitak-bilgem-yte/ember-contextual-table#29]
I need to be able to set a data attribute on a row such as: `<tr data-foo="bar">`.  is there any straightforward way to do this right now?

**Comment by ykaragol** [comment]
Nope. Table is rendering `tr` tags directly and there is no ember way to do this for now. But you can use dom modification (such as jquery).

**Comment by jelhan** [comment]
This is a huge limitation for testing IMO. Normally I would add a data attribute as test selector on each table row, which could be used in tests to assert that a specific record is shown in the list. E.g. a simple test might look like the following:

```js
let [user] = this.server.createList('user', 10); // create a user in mirage
await visit('/users'); // open a page that lists users on a table
assert.dom(`[data-test-table="users"] [data-test-user="${user.id}] [data-test-column="name"]`)
  .hasText(`${user.firstName} ${user.lastName}`);
```

Not being able to select the row for a specific record makes testing much more difficult. :disappointed:
